c-log/README.md

```diff
@@ -0,0 +1,188 @@
+# clog
```

Comment:
I wanted to make it distinct from the standard logger which is already using `log` and `logger` so I'm using c-log for clever log. Let me know if another name is more appropriate.